pkg/agent/config/config.go

```diff
@@ -475,6 +475,7 @@ func get(envInfo *cmds.Agent) (*config.Node, error) {
 	nodeConfig.AgentConfig.DisableCCM = controlConfig.DisableCCM
 	nodeConfig.AgentConfig.DisableNPC = controlConfig.DisableNPC
 	nodeConfig.AgentConfig.Rootless = envInfo.Rootless
+	nodeConfig.AgentConfig.Manifests = filepath.Join(envInfo.DataDir, "staticpods")
```

Comment:
We need some better naming here. `Manifests` in the `AgentConfig` slightly matches up in spirit with the kubelet `pod-manifest-path` parameter but it has unfortunate overlap with the established k3s concept of the manifest directory for deploying things other than static pods. Maybe `StaticManifests`?

Reply:
What about pod manifests?  I think my bigger concern is what should the directory name should be called. Do you have a suggests or is "staticpods" fine.  We already have "server/manifests" and "agent/manifests" seems like it would be confusing, but maybe not?  I could do "agent/podmanifests"

Reply:
I like `agent/manifests` because if the kublet is ever changed via a KEP or the nomenclature otherwise changes such that they aren't called static pods but the location is still used to deploy "manifests" it will still make sense. If we want to avoid being bit by that bit of slickness then let us be more explicit and adopt something like `agent/static-pod-manifests`.

Reply:
I ended up going with `pod-manifests` as that corresponds to `--pod-manifest-path`